modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/SAMLMetadataTestCase.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.integration.test.saml;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.wso2.carbon.tenant.mgt.stub.TenantMgtAdminServiceExceptionException;
+import org.wso2.identity.integration.common.utils.ISIntegrationTest;
+import org.wso2.identity.integration.test.utils.DataExtractUtil;
+import org.json.JSONObject;
+import org.json.XML;
+
+import java.io.IOException;
+
+public class SAMLMetadataTestCase extends ISIntegrationTest {
+
+    private static final String SAML_METADATA_ENDPOINT_SUPER_TENANT = "https://localhost:9853/identity/metadata/saml2";
+    private static final String SAML_METADATA_ENDPOINT_TENANT =
+            "https://localhost:9853/t/wso2.com/identity/metadata/saml2";
+    private static final String SAML_METADATA_ENDPOINT_WITH_SUPER_TENANT_AS_PATH_PARAM =
+            "https://localhost:9853//t/carbon.super/identity/metadata/saml2";
```

Comment:
```suggestion
            "https://localhost:9853/t/carbon.super/identity/metadata/saml2";
```

Reply:
Fixed https://github.com/wso2/product-is/pull/8319/commits/615aad369e02f588fa1836383528faea0b3ca014